corehq/apps/app_manager/models.py

```diff
@@ -1005,7 +1007,7 @@ def default_name(self):
         app = self.get_app()
         return trans(
             self.name,
-            [app.default_language] + app.build_langs,
+            [app.default_language] + app.langs,
```

Comment:
It looks like its mostly used to show form and module names in HQ and not during build. The one place I found it during a build is in a validation error message, that is shown in HQ as well. I think because of that it makes sense to leave this as is. I corrected use of default_language where it is used